test/test_jit.py

```diff
@@ -950,6 +950,38 @@ def test_torch_autograd_backward_with_grad_tensors(input):
         checkBackwardScript(test_torch_autograd_backward, (inp,))
         checkBackwardScript(test_torch_autograd_backward_with_grad_tensors, (inp,))
 
+    def test_script_backward_twice(self):
+        def checkBackwardTwiceScript(fn, inputs, retain_graph_=False):
+            torch._C._jit_set_profiling_executor(False)
```

Comment:
OOC, why do you choose to set the executor state here without restoring it to previous value?

I noticed that this test interferes with one of tests that I am trying to add. Would it be better if we only run this test when Profiling Executor is turned off, using @unittest.skipIf?

Reply:
I can skip it when profiling executor is turned on. Are the tests run with profiling executor off under some testsuite already?

Reply:
In that case, I think you need to decorate your test case with

@unittest.skipIf(GRAPH_EXECUTOR == ProfilingMode.LEGACY, "REASON REASON REASON")

Reply:
Raised https://github.com/pytorch/pytorch/pull/45893